From 822b8d1ea6001c42c6179f62efda2ec74093fffd Mon Sep 17 00:00:00 2001
From: Daniel Zhang <danielzhang130@gmail.com>
Date: Fri, 16 Dec 2022 12:12:59 -0500
Subject: [PATCH] Remove ununsed apps

Signed-off-by: Daniel Zhang <danielzhang130@gmail.com>
---
 products/gms.mk | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/products/gms.mk b/products/gms.mk
index bfdb8f6..0796a27 100644
--- a/products/gms.mk
+++ b/products/gms.mk
@@ -61,16 +61,7 @@ PRODUCT_PACKAGE_OVERLAYS += $(ANDROID_PARTNER_GMS_HOME)/overlay/gms_overlay
 
 # GMS mandatory application packages
 PRODUCT_PACKAGES += \
-    Chrome \
-    Drive \
-    Gmail2 \
-    Duo \
-    Maps \
-    YTMusic \
-    Photos \
     Velvet \
-    Videos \
-    YouTube
 
 # GMS comms suite
 $(call inherit-product, $(ANDROID_PARTNER_GMS_HOME)/products/google_comms_suite.mk)
@@ -81,26 +72,16 @@ $(call inherit-product, $(ANDROID_PARTNER_GMS_HOME)/products/personal_safety.mk)
 # GMS optional application packages
 PRODUCT_PACKAGES += \
     AndroidSystemIntelligence_Features \
-    CalendarGoogle \
     DeskClockGoogle \
     LatinImeGoogle \
     TagGoogle \
     talkback \
-    Keep \
-    CalculatorGoogle
 
 # Device Health Services
 $(call inherit-product, $(ANDROID_PARTNER_GMS_HOME)/products/turbo.mk)
 
-# GMS sample application packages
-PRODUCT_PACKAGES += \
-    GmsSampleIntegration \
-    SearchLauncherQuickStep
-
 include $(ANDROID_PARTNER_GMS_HOME)/products/gms_package_version.mk
 
 PRODUCT_PRODUCT_PROPERTIES += \
-    ro.setupwizard.rotation_locked=true \
-    setupwizard.theme=glif_v3_light \
     ro.opa.eligible_device=true \
     ro.com.google.gmsversion=$(GMS_PACKAGE_VERSION_ID)
-- 
2.34.1

